pkg/servicescm/servicescm.go

```diff
@@ -0,0 +1,21 @@
+package servicescm
```

Comment:
For simplicity, how about referring to this just as `services` in the package name, commit message etc?